app-text/unrtf/unrtf-0.21.9.recipe

```diff
@@ -0,0 +1,52 @@
+SUMMARY="Rich Text Format command line converter"
+DESCRIPTION="UnRTF is a command-line program written in C which converts \
+documents in Rich Text Format (.rtf) to HTML, LaTeX, troff macros, and RTF \
+itself."
+HOMEPAGE="https://www.gnu.org/software/unrtf/"
+COPYRIGHT="2001-2015 Free Software Foundation"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://www.gnu.org/software/unrtf/unrtf-$portVersion.tar.gz"
+CHECKSUM_SHA256="22a37826f96d754e335fb69f8036c068c00dd01ee9edd9461a36df0085fb8ddd"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```

Comment:
I confess that I skipped testing it on x86_gcc2. Could you explain what this change does? Why change the binary directory?